tools/run-fbp-tests

```diff
@@ -44,6 +44,104 @@ import re
 import subprocess
 import sys
 
+valgrind_opts = [
```

Comment:
Use a tuple if these values aren't supposed to be changed. While there, rename to `VALGRIND_OPTS` so it screams it's a constant.